### PR TITLE
Nutanix steps and chain for config and deprovision

### PR DESCRIPTION
**Pull request by adiantum**
- CI chain and steps added for Nutanix IPI conf
    and deprovision
  - conf step for obtaining Nutanix specific params
  - conf steps for DNS records add and delete in AWS
  - conf step for install-config

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/nutanix/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
+- staebler
+- jcpowermac
+- patrickdillon
+- jstuever
+- rvanderp3
```
Comment (thunderboltsid) [inline ci-operator/step-registry/ipi/conf/nutanix/OWNERS]: shouldn't this be Nutanix reviewers and approvers?

File: ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+}]}
+EOF
+
+# api-int record is needed for Windows nodes
+# TODO: Remove the api-int entry in future
+echo "Creating batch file to destroy DNS records"
+
+cat > "${SHARED_DIR}"/dns-delete.json <<EOF
+{
+"Comment": "Delete public OpenShift DNS records for VSphere IPI CI install",
```
Comment (thunderboltsid) [inline ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh]: vSphere?

File: ci-operator/step-registry/ipi/conf/nutanix/OWNERS (hunk 1 above)
Comment (adiantum) [reply ci-operator/step-registry/ipi/conf/nutanix/OWNERS]: I've checked several cloud and looks like there is the same RedHat people are owners. We could ask them.

File: ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh (hunk 2 above)
Comment (adiantum) [reply ci-operator/step-registry/ipi/conf/nutanix/dns/ipi-conf-nutanix-dns-commands.sh]: Good catch, thank you. This part actually taken from vsphere implementation.

File: ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+baseDomain: ${BASE_DOMAIN}
+platform:
+  nutanix:
+    apiVIP: ${API_VIP}
+    ingressVIP: ${INGRESS_VIP}
+    insecure: true
+    password: ${NUTANIX_PASSWORD}
+    port: ${NUTANIX_PORT}
+    prismCentral: ${NUTANIX_HOST}
+    prismElementUuid: ${PE_UUID}
```
Comment (thunderboltsid) [inline ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh]: s/Uuid/UUID/g

Comment (adiantum) [reply ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh]: Why? 
In https://github.com/nutanix-cloud-native/openshift-installer/blob/86096fbbbdc5238673f1b43a91ca0d9ce8e3d3d9/pkg/types/nutanix/platform.go#L21 this param have struct tag `json:"prismElementUuid"`

Comment (thunderboltsid) [reply ci-operator/step-registry/ipi/conf/nutanix/ipi-conf-nutanix-commands.sh]: Yeah, they made us change that on the public PR https://github.com/openshift/installer/pull/5603/files#diff-8ef64117e2777ca1c952e4de34ccdde491181b070a5b104e99cadfc15fb2d22bR18

**Pull request merged**